x-pack/legacy/plugins/ml/public/data_frame/components/source_index_preview/source_index_preview.tsx

```diff
@@ -248,6 +248,22 @@ export const SourceIndexPreview: React.SFC<Props> = React.memo(({ cellClick, que
             <EuiBadge>array</EuiBadge>
           </EuiToolTip>
         );
+      } else if (typeof d === 'object' && d !== null) {
+        // If the cells data is an object, display a 'object' badge with a
+        // tooltip that explains that this type of field is not supported in this table.
+        return (
+          <EuiToolTip
+            content={i18n.translate(
+              'xpack.ml.dataframe.sourceIndexPreview.dataFrameSourceIndexObjectToolTipContent',
+              {
+                defaultMessage:
+                  'The full content of this object based column is available in the expanded row.',
+              }
+            )}
+          >
+            <EuiBadge>object</EuiBadge>
```

Comment:
Maybe safest to internationalize the word `object`, and `array` in the block above?

Reply:
Fixed in e3f50d5.